src/Migrations/Version20240319121423.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\PayPalPlugin\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Sylius\Bundle\CoreBundle\Doctrine\Migrations\AbstractPostgreSQLMigration as SyliusAbstractMigration;
+
+if (class_exists(SyliusAbstractMigration::class)) {
```

Comment:
https://github.com/Sylius/Sylius/blob/1.13/UPGRADE-1.13.md#miscellaneous
```
PostgreSQL migration support has been introduced. If you are using PostgreSQL, we assume that you have already created a database schema in some way. All you need to do is run migrations, which will mark all migrations created before Sylius 1.13 as executed.
```
I believe we should handle new migrations by creating them for both PostgreSQL and MySQL and ensure that the old ones can be safely marked as executed, similar to what we did in Sylius:
https://github.com/Sylius/Sylius/blob/4f499b5ff7729936da37853b276a730a1d4afb16/src/Sylius/Bundle/CoreBundle/Migrations/Version20230419092354.php#L23

Reply:
I've added code to mark migration as executed